src/components/drag-and-drop/drag-and-drop.stories.js

```diff
@@ -54,21 +59,21 @@ export const Default = () => {
   return (
     <DraggableContext autoScroll={autoScroll} onDrag={handleDrag}>
       <div>
-        <Table tbody={false}>
+        <FlatTable tbody={false}>
           <thead>
-            <TableRow as="header">
-              <TableHeader />
-              <TableHeader>Country</TableHeader>
-            </TableRow>
+            <FlatTableRow as="header">
```

Comment:
`as` prop does not exist on `FlatTableRow` (remove this from other files as well